broker-daemon/orderbook/index.js

```diff
@@ -150,6 +150,36 @@ class Orderbook {
       sequence
     }
   }
+
+  /**
+   * Gets current orderbook events by timestamp
+   *
+   * @param {String} timestamp - timestamp in nano-seconds
+   * @returns {Array<MarketEventOrder>}
+   */
+  async getOrderbookEventsByTimestamp (timestamp) {
+    return getRecords(
+      this.store,
+      (key, value) => JSON.parse(value),
+      // Limits the query to gte to a specific timestamp
+      MarketEvent.rangeFromTimestamp(timestamp)
```

Comment:
When do we use a MarketEvent vs a MarketEventOrder?

Reply:
MarketEvent is data that comes from the relayer. When we add the information to the orderbook, and then subsequently search for it, we serialize that data to MarketEventOrder.

```
MarketEvent = events from relayer
MarketEventOrder = anything on the orderbook (sent to the client)
```

ill add this information to each class. Additionally, the store that is being used in this case, is `eventStore` which is `MarketEvents`, where as the orderbookStore is `MarketEventOrders`